src/xradio/vis/_vis_utils/_ms/partition_queries.py

```diff
@@ -74,10 +74,10 @@ def create_partitions(in_file: str, partition_scheme: Union[str, list], vla_otf=
         )
         if source_tb.nrows() != 0:
```

Comment:
Just noticed that these main_tb, field_tb, source_tb, state_tb are leaked (never closed here I think?). It would be safer to use the usual open_table_ro or similar. But that can be fixed in the partition_queries maintenance issue (https://github.com/casangi/xradio/issues/192 )